src/components/ASPerformance.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PlotlyGraph from './Plot.js';
+import Select from 'react-select';
+import Highlighter from 'react-highlight-words';
+import 'react-select/dist/react-select.css';
+import '../css/temp.css' //this is temp import - needs to be removed for bundle
+
+import { fetchAsDataIfNeeded, AsIsSelected } from '../actions/ASactions';
+
+export class ASPerformance extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      plotlyData: [],
+      graphOptions: {
+        legend: {x:0, y:1},
+        height: 200,
+        margin: {
+          l: 30,r: 30,
+          b: 30,t: 0
+        },
+        font: {
+          size: 9,
+          color: '#7f7f7f'
+        }
+      }
```

Comment:
I think these options should not be in the Redux store because each page has its own layout and defining them inside component is easier to maintain. And there is no need to pass them from the server things like height, margin etc.